### DISCUSSION TITLE
SingalR server setup in NodeJs Application

### DISCUSSION THREAD

**Discussion by MathavanN** [dotnet/aspnetcore#28223] [Q&A] [answered]
I have a NodeJs API application. Is it possible to setup singalr HUB setup in my NodeJs API application?

I tried the below setup
let connection = new signalR.HubConnectionBuilder()
    .withUrl("/chat")
    .build();

Error: Cannot resolve '/chat'.

**Comment by BrennanConroy** [comment, marked as answer]
You need to give a full url in a NodeJs app. For example "http://localhost:5000/chat"

**Reply by davidfowl** [reply]
Are you trying to write a nodejs client with a .NET Server? SignalR has only been officially implemented on the server side with .NET.

**Reply by MathavanN** [reply]
@davidfowl I have a NodeJS API app, an Angular client app. I need to set up a SignalR in NodeJS app.

**Reply by davidfowl** [reply]
Then when @BrennanConroy says is relevant
